packages/react/src/auto/mui/inputs/MUIAutoJSONInput.tsx

```diff
@@ -11,24 +11,23 @@ export const MUIAutoJSONInput = (
     control?: Control<any>;
   } & Partial<TextFieldProps>
 ) => {
-  const { onStringValueChange, error, stringValue, originalController } = useJSONInputController(props);
   const [isFocused, focusProps] = useFocus();
   const { field: _field, control: _control, ...restOfProps } = props;
+  const { type: _type, errorMessage, ...controller } = useJSONInputController(props);
 
-  const inErrorState = !isFocused && !!error;
+  const inErrorState = !isFocused && !!errorMessage;
 
   return (
     <TextField
       multiline
       maxRows={4}
       inputProps={{ style: { fontFamily: "monospace" } }}
-      {...originalController}
-      value={stringValue}
-      onChange={(event) => onStringValueChange(event.target.value)}
       error={inErrorState}
-      helperText={inErrorState && `Invalid JSON: ${error.message}`}
+      helperText={inErrorState && `Invalid JSON: ${errorMessage}`}
+      {...controller}
       {...focusProps}
       {...restOfProps}
+      onChange={(event) => controller.onChange(event.target.value)}
```

Comment:
An `onChange` passed into this component will not run because of this. Perhaps this should be 
```
    onChange={(event) => {
        controller.onChange(event.target.value);
        props.onChange?.(event);
      }}
```

Reply:
Is this something we're trying to keep support for across the board?

Reply:
The `onChange` would just need to go up a few lines to allow for users to override it. I don't see any onChange or onClick examples where we do the onChange from the controller as well as the passed in onChange prop. Although I realize now that some people may want to completely avoid the default behaviour and we should allow for that

Reply:
As discussed in slack we're just not gonna expose `onChange` for now since its not quite clear if it should be an override or a participant or if a user-supplied one can influence ours 👍